packages/fxa-payments-server/package.json

```diff
@@ -62,17 +64,28 @@
     "@types/storybook__react": "^4.0.1",
     "@typescript-eslint/eslint-plugin": "^1.9.0",
     "@typescript-eslint/parser": "^1.9.0",
+    "async-wait-until": "^1.2.4",
     "audit-filter": "^0.5.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
     "browserslist": "^4.7.2",
     "caniuse-lite": "^1.0.30001010",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "chai-enzyme": "^1.0.0-beta.1",
+    "enzyme": "3.7.0",
```

Comment:
I don't think we want to add enzyme & chai - we're already using jest & react-testing-library as a framework here

Reply:
I agree, pulled the l10n and l10.test.js files from `lockwise-addon`. Wanted to ensure it all worked before converting to jest/react-testing-library